docs/logging.md

```diff
@@ -0,0 +1,56 @@
+## Logging Congiguration
+--
+
+firecracker-containerd allows for users to specify per application/library
+logging. We do this by setting the `log_level` field in the
+firecracker-runtime.json file.
+
+```json
+{
+  "firecracker_binary_path": "/usr/local/bin/firecracker",
+  "kernel_image_path": "/var/lib/firecracker-containerd/runtime/default-vmlinux.bin",
+  "kernel_args": "ro console=ttyS0 noapic reboot=k panic=1 pci=off nomodules systemd.journald.forward_to_console systemd.unit=firecracker.target init=/sbin/overlay-init",
+  "root_drive": "/var/lib/firecracker-containerd/runtime/default-rootfs.img",
+  "cpu_count": 1,
+  "cpu_template": "T2",
+  "log_levels": "debug",
+  "jailer": {
+    "runc_binary_path": "/usr/local/bin/runc"
+  }
+}
+```
+
+| log levels                     | description                                                               |
+| :---------------------------:  | :-----------------------------------------------------------------------: |
+| debug                          | This will set all log levels to debug                                     |
+| firecracker_error              | Logs any error information in Firecracker's lifecycle                     |
```

Comment:
How about using `=` or `:` as the separator, instead of `_`?

Reply:
Yea, I thinking about using `:` but ultimately went just with `_`. But since you mentioned it, I'll go ahead and switch it to that.